src/api/json/catalog.json

```diff
@@ -1091,6 +1091,7 @@
       "name": "AWS CloudFormation Serverless Application Model (SAM)",
       "description": "The AWS Serverless Application Model (AWS SAM, previously known as Project Flourish) extends AWS CloudFormation to provide a simplified way of defining the Amazon API Gateway APIs, AWS Lambda functions, and Amazon DynamoDB tables needed by your serverless application.",
       "fileMatch": [
+        "template.yaml",
```

Comment:
This seems a bit too generic and will result in false positive matches. Are template.yaml files typically in certain folders that you can add to the globbing pattern?

Reply:
No, unfortunately, it can be created anywhere. It's common to have it in the project's root folder.

I know the file name `template.yaml` sounds very vague, but I think AWS SAM template is the only schema that uses it. I also searched the catalog to try finding any matches to `template.yaml` but found none.

Reply:
Ok, let's give it a go